.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
```

Comment:
Nice work adding the `linter.yml` file in this folder path. However, I advise that you follow **all** the instructions (from top to bottom) on [this page](https://github.com/microverseinc/linters-config/tree/master/html-css) completely, to be able to effectively track all coding anomalies. 😀